coops-ui/src/mixins/common-mixin.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Vue } from 'vue-property-decorator'
+import omit from 'lodash.omit'
+
+/**
+ * Mixin that provides some useful address utilities.
```

Comment:
Not 'address' :-P

If you're using this mixin to 'type' some methods, do you think we could use this instead?
http://definitelytyped.org/
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/json-schema/index.d.ts

(No change required in code above.)

Reply:
:) Good catch, thank you!  I'll have a read and get back to you

Reply:
Done.